### PR TITLE
Feature/optional module map

### PR DESCRIPTION
**Pull request by JamesSingleton**

### CODE REVIEW

**Comment by anescobar1991** [comment]
Did you validate this locally? Also can you add a description for transparency for others?

File: packages/one-app-runner/bin/one-app-runner.js [hunk 1]
```diff
@@ -72,6 +71,8 @@ const createYargsConfig = () => {
     })
     .option('modules', {
       type: 'array',
+      // eslint-disable-next-line global-require
+      demandOption: !require('yargs').argv.moduleMapUrl,
```
Comment (anescobar1991) [inline packages/one-app-runner/bin/one-app-runner.js]: Can you add a test?

**Comment by JamesSingleton** [comment]
> Did you validate this locally? Also can you add a description for transparency for others?

Yes, I ran the bin with options and validated that if `--module-map-url` is not provided that `modules` is and visa verse.

File: packages/one-app-runner/bin/one-app-runner.js (hunk 1 above)
Comment (JamesSingleton) [reply packages/one-app-runner/bin/one-app-runner.js]: Technically this test is already covering that.
```
test('command errors out if --module-map-url option is not given', () => {
  const consoleErrorSpy = jest.spyOn(console, 'error');

  process.argv = ['', '', '--root-module-name', 'frank-lloyd-root', '--docker-image', 'one-app:5.0.0'];
  jest.mock('../../src/startApp', () => jest.fn());
  require('../../bin/one-app-runner');
  expect(consoleErrorSpy.mock.calls).toMatchSnapshot();
});
```

But I can add something like
```
test('--modules option is required if --module-map-url option is not given', () => {
  const consoleErrorSpy = jest.spyOn(console, 'error');

  process.argv = ['', '', '--root-module-name', 'frank-lloyd-root', '--docker-image', 'one-app:5.0.0'];
  jest.mock('../../src/startApp', () => jest.fn());
  require('../../bin/one-app-runner');
  expect(consoleErrorSpy.mock.calls).toMatchSnapshot();
});
```

Comment (JamesSingleton) [reply packages/one-app-runner/bin/one-app-runner.js]: Added in https://github.com/americanexpress/one-app-cli/pull/48/commits/e0828338471fc7749b49630e11e78688e7b29967

File: packages/one-app-runner/__tests__/bin/one-app-runner.spec.js [hunk 2]
```diff
@@ -49,6 +49,8 @@ test('command errors out if --module-map-url option is not given a value', () =>
 test('command errors out if --module-map-url option is not given', () => {
   const consoleErrorSpy = jest.spyOn(console, 'error');
 
+  console.log('Console Error: ', consoleErrorSpy.mock.calls);
```
Comment (anescobar1991) [inline packages/one-app-runner/__tests__/bin/one-app-runner.spec.js:52]: Did you mean to leave this in?

**Pull request merged**